cli/src/deploy/mod.rs

```diff
@@ -13,6 +13,10 @@ use std::path::Path;
 use ascii_table::{AsciiTable, Column};
 use base64;
 
+const REQUEST_TIMEOUT: &str = "Sorry, the cloud provider didn't get back to us in the expected time. \
```

Comment:
Let's just do `Operation is still in progress. It might take a few more minutes for the cloud provider to finish up.` We don't have a status column in `info` yet.